File: omnibook-broker/src/main/java/com/sprint/omnibook/broker/translator/TranslationException.java
package com.sprint.omnibook.broker.translator;

/**
 * Payload 변환 실패 시 발생하는 예외.
 */
public class TranslationException extends RuntimeException {

    public TranslationException(String message) {
        super(message);
    }

    public TranslationException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: omnibook-broker/src/main/kotlin/com/sprint/omnibook/broker/translator/TranslationException.kt
package com.sprint.omnibook.broker.translator

/**
 * Payload 변환 실패 시 발생하는 예외.
 */
class TranslationException : RuntimeException {
    constructor(message: String) : super(message)
    constructor(message: String, cause: Throwable) : super(message, cause)
}
